Sources/MapboxCoreNavigation/EHorizon/EHorizonObjectEdgeLocation.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import MapboxNavigationNative
+
+public struct EHorizonObjectEdgeLocation {
+    /**
+     Offset from the start of edge (0 - 1) pointing to the beginning of road object on this edge
+     will be 0 for all edges in the line-like road object expect the very first one
+     in the case of point-like object percentAlongBegin == percentAlongEnd
```

Comment:
I understand that this is just the copy-pasted description from `RoadObjectEdgeLocation`, but I've read it several times and still didn't understand what it has to do with line-like road objects. I believe there are few typos in there :)

Reply:
That's just a typo: expect = except.